src/RegionsContainer.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+
+// import { getRegions } from './services/api';
+
+function RegionsContainer() {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch({ type: 'GET_REGIONS' });
+  }, []);
+  return (
+    <div className="regions-container">RegionsContainer</div>
+  );
```

Comment:
```suggestion
function RegionsContainer() {
  const dispatch = useDispatch();
  
  useEffect(() => {
    dispatch({ type: 'GET_REGIONS' });
  }, []);
  
  return (
    <div className="regions-container">RegionsContainer</div>
  );
```

의미상으로 구분되는 경우 빈 줄을 넣어주면 코드 가독성에 더 도움이 될 것 같아요!